layouts/partials/contributors.html

```diff
@@ -0,0 +1,146 @@
+  The data looks like:
+  ```js
+  [
+    {"url":"https://github.com/0intro","photo":"https://avatars.githubusercontent.com/u/6043744?v=3","username":"0intro"}
+  ]
+  ```
+*/}}
+{{ $contributors := getJSON "https://contributors.cloud.ipfs.team/contributors?org=all" }}
+<script>
+  (function () {
+    var contributors = {{ $contributors }}
+    var hexWidth = 62
+    var hexHeight = 72
+    var loadingDelayMs = 150
+    var containerEl = document.getElementById('contributors-slide')
+    var containerElTop = containerEl.getBoundingClientRect().top
+    var screenHeight = window.innerHeight
+    var pageYOffset = window.pageYOffset
+    var inView = (pageYOffset + screenHeight) > containerElTop
+    if (inView) {
+      renderContributors(contributors)
+    } else {
+      window.addEventListener('scroll', isContributorsSectionInView)
+    }
+
+    function isContributorsSectionInView () {
+      inView = (window.pageYOffset + screenHeight) > containerElTop
+      if (inView) {
+        renderContributors(contributors)
+        window.removeEventListener('scroll', isContributorsSectionInView)
+      }
+    }
+
+    function renderContributors (data) {
+      containerEl.className = 'running'
+      var rows = document.querySelectorAll('.contributors-row')
+      var rowWidth = Math.ceil(data.length / rows.length) * hexWidth
+      ;[].forEach.call(rows, function (row, i) {
+        row.style.height = hexHeight + 'px'
+        row.style.width = rowWidth + 'px'
+        row.style.top = i * hexHeight * 0.75 + 'px'
+        if ((i % 2) === 1) {
+          row.style.left = hexWidth / 2 + 'px'
+        }
+      })
+      var nextRow = 0
+      function renderWithPause(itemIndex, lastRender) {
+        renderContributor(data[itemIndex], function (err, el) {
+          rows[nextRow].appendChild(el)
+          setTimeout(function () {
+            el.className = 'hex show'
+          }, 10)
+          nextIndex = itemIndex + 1
+          if (nextIndex === data.length) return // All done.
+          nextRow = (nextRow + 1) % rows.length
+          var now = Date.now()
+          var diff = now - lastRender
+          var delay = loadingDelayMs - diff
+          if (delay < 0) delay = 0
+          setTimeout(function () {
+            renderWithPause(itemIndex + 1, now)
+          }, delay)
+        })
+      }
+      renderWithPause(0, Date.now())
+    }
+
+    function renderContributor (item, cb) {
+      var link = document.createElement('a')
+      link.className = 'hex'
+      link.href = 'https://github.com/' + item.username
+      link.target = '_blank'
+      link.title = item.username
+      link.style.width = hexWidth + 'px'
+      link.style.height = hexHeight + 'px'
+
+      var img = new Image()
+      img.onload = function () {
+        link.style.background = '#333 url(' + item.photo + ') no-repeat center'
+        link.style.backgroundSize = 'cover'
+        cb(null, link)
+      }
```

Comment:
If the image fails to load you halt the animation I think. You need an `onerror` handler to call the callback in this case.